vpt/processors/video_engagement_estimator/__init__.py

```diff
@@ -0,0 +1,36 @@
+from rx import Observable
+from rx.subject import Subject
+
+from vpt.processors.base import ProcessorBase
+from vpt.sources.base import SourceBase
+
+import numpy as np
+
+
+class VideoEngagementEstimator(ProcessorBase):
+    """Returns:
+        - none if no face was detected in the video frame
+        - true if face was detected and it is looking at the camera
+        - false if face was detected but it looks sideways
```

Comment:
So you didn't just use Boolean to represent a value with 2 states that is not related to truthiness, you use a nullable boolean to represent 3 states!
I did not want to comment on code quality, but come on..

Reply:
This is how it is made in our state machine, that's not my decision

Reply:
I am referring to the use of the boolean variable, not the states. Like, use an `enum` or something